docs/sdk/features.md

```diff
@@ -302,30 +302,39 @@ For instructions on setting up a custom domain, see [Custom Domains](/platform/c
 
 ## Realtime Updates
 
-DevCycle SDKs are capable of being notified in real time that new configuration changes have been made in the DevCycle platform.
+DevCycle SDKs (both client-side and server-side) are capable of being notified in realtime that new configuration changes have been made in the DevCycle platform.
 DevCycle leverages Server-Sent Events (SSE) to notify the SDKs that a feature (targeting rules, variable values, etc.)
-has been saved and that they should fetch the new configuration.
+has been saved and that they should fetch the new configuration. A connection URL is included in the config that the SDK fetches, 
+triggering the SDK to open a connection with our SSE provider listening for any changes in the given environment.
+
+**Supported SDKs**
+- Javascript
+- React
+- iOS
+- Android
+- Flutter
+- NodeJS (including Next and Nest)
+- Python
+- Ruby
+- Java
+- Go
+- .NET
+- PHP (via SDK Proxy)
+
+### SDK Specifics
+
+#### **Javascript SDK**, **React SDK**
 
-#### Client-Side SDK
-
-A connection URL is included in the config that the SDK fetches, triggering the SDK to open a connection with our SSE provider listening
-for any changes from the dashboard.
-
-The following Client-Side SDKs currently have Realtime Updates:
+If the user loses focus on the webpage for longer then the `inactivityDelay` (the default of which is set to 2 minutes, and can be configured through the options), the SDK will disconnect from the SSE provider and will reconnect when the user opens the tab / window again (i.e. the page's visibility state = `visible`). The SDK will also request a new configuration during reconnection to receive any updates it may have missed while the realtime connection was closed.
 
-- Javascript SDK
-- React SDK
-- iOS SDK
-- Android SDK
-- Flutter SDK
+#### **iOS SDK**, **Android SDK** & **Flutter SDK**
 
-##### **Javascript SDK**, **React SDK**
+If the user backgrounds the application for some period of time, the SDK will disconnect from the SSE provider and will reconnect again when the user brings the application to the foreground. When the application is brought to the foreground the SDK will request a new configuration to receive any updates it may have missed while the realtime connection was closed.
 
-If the user loses focus on the webpage for longer then the `inactivityDelay` (the default of which is set to 2 minutes, and can be configured through the options), the SDK will disconnect from the SSE provider and will reconnect when the user opens the tab / window again (i.e. the page's visibility state = `visible`). The SDK will also request a new configuration during reconnection to receive any updates it may have missed while the realtime connection was closed.
+#### Server-Side SDKs
 
-##### **iOS SDK**, **Android SDK** & **Flutter SDK**
+If the server loses its connection it will be re-opened on the next config poll, which can be configured as desired.
```

Comment:
what would "next config poll" mean to someone reading this?

Reply:
given it's client side, is next initialization better?

Reply:
ok edited